lib/screens/play/view_discard_page.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+import 'package:treasure_of_the_high_seas/model/view_mode.dart';
+import '../../model/card/card.dart' as Model;
+import 'card_viewer.dart';
+
+class ViewDiscardPage extends StatelessWidget {
+  final GameState state;
+
+  ViewDiscardPage(this.state);
+
+  @override
+  Widget build(BuildContext context) {
+    final discard = context.select<GameState, List<Model.Card>>((state) => state.discard);
+
+    return CardViewer(discard,
+        ViewMode.DISCARD, closeButton);
+  }
+
+  List<Widget> closeButton(context, card){
+    return [_getCloseButton(context, card)];
```

Comment:
This current approach totally works, so feel free to do this as a separate change (if at all):

In my head I'd imagined we'd pass an empty list to the `CardViewer` and render our own close button - it'd give us more control over exactly where we want it / how to style it etc. Thinking that a static exit button might make more sense, given that the action isn't tied into the card currently being viewed in any way.

Reply:
Ok, I'll raise a separate ticket for that, I agree that the close button looks bad swiping with everything.

Reply:
Raised as https://github.com/kilikfan/treasure/issues/71